exec/java-exec/src/main/java/org/apache/drill/exec/work/metadata/ServerMetaProvider.java

```diff
@@ -76,7 +76,7 @@
       .setReadOnly(false)
       .setGroupBySupport(GroupBySupport.GB_UNRELATED)
       .setLikeEscapeClauseSupported(true)
-      .setNullCollation(NullCollation.NC_AT_END)
+      .setNullCollation(NullCollation.NC_HIGH)
```

Comment:
I'm not sure if the last sentence is correct. If user specified DESC, and null collation is unspecified, then null should sort first, no (if always HIGH). NC_HIGH seems then the correct value (unless the default can be changed in the planner configuration).